### PR TITLE
graphql playground middleware

### PR DESCRIPTION
**Pull request by viebel**
Add a middleware to server another graphql IDE, namely [https://github.com/prisma-labs/graphql-playground/](graphql-playground)

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -8,5 +8,6 @@
                  [metosin/ring-http-response "0.9.0"]
                  [ring/ring-core "1.6.0"]
                  [threatgrid/ring-graphiql "0.1.1"]
-                 [threatgrid/ring-graphql-voyager "0.1.2"]]
-  :profiles {:test {:dependencies [[ring/ring-mock "0.3.0"]]}})
+                 [viebel/ring-graphql-playground "0.1.1"]]
+  :profiles {:test {:dependencies [[ring/ring-mock "0.3.0"]]}
+             :dev {:dependencies [[ring/ring-mock "0.3.0"]]}})
```
Comment (vemv) [inline project.clj]: Better to leave this untouched and run the repl w/ `lein with-profile +test repl`

**Review by vemv — COMMENTED** [review]
Interesting PR thanks!

At the same time, introducing a functionality for a feature we don't use is sort of a vulnerability vector - it's code that will be reviewed less, possibly allowing arbitrary changes because "we don't use it anyway".

As a (non-authoritative) suggestion, maybe you can port this code to your own lib, which we'd be happy to link from in the README.

**Comment by viebel** [comment]
@vemv On one hand I understand that you don't want to add functionality that you don't use to your code base.  On the other hand, you are not the only user of your repo.

I am fine with porting the addition of the playground IDE to my own lib.  I see two options:
1. Including in this lib the original 2 IDEs + playground
2. Including only playground

What I don't like with option 1 is that it might cause some users to use my lib instead of yours.
What I don't like with option 2 it that users who want the 3 IDEs need to include 2 dependencies.

My preference would be that you add the playground IDE (and maybe upcoming IDEs) in your original repo.

What do you guys think?

File: project.clj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-(defproject threatgrid/ring-graphql-ui "0.1.3-SNAPSHOT"
+(defproject threatgrid/ring-graphql-ui "0.1.3"
```
Comment (msprunck) [inline project.clj]: Please do not remove the SNAPSHOT prefix, I remove it for new releases only

File: src/ring_graphql_ui/core.clj [hunk 3]
```diff
@@ -94,7 +94,7 @@
 
 (defn voyager
   "Returns a Ring handler which can be used to serve GraphQL Voyager"
-  ([] (graphiql {}))
+  ([] (voyager {}))
```
Comment (msprunck) [inline src/ring_graphql_ui/core.clj:97]: good catch!

**Review by msprunck — COMMENTED** [review]
Thank you for this addition! Once the comments are taken into account, I will merge it and release a new version

**Comment by viebel** [comment]
@msprunck All comments addressed

**Comment by msprunck** [comment]
> @msprunck All comments addressed

Awesome! Thank you for your contribution @viebel! 0.1.3 has been deployed on clojars

**Pull request merged**